files/logstash-configs/23-ufw.conf

```diff
@@ -15,4 +15,17 @@ filter {
       }
     }
   }
+  if [type] == "ufw" {
```

Comment:
Rather than `if`, this can be `else if`, since the two conditions are mutually exclusive.